e2e_tests/tests/per_provider/normal_tests/key_agreement.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Contributors to the Parsec project.
```

Comment:
For raw key agreement can you have a test where you generate two keys pairs, do the raw agreement on each private key with the other key pair's public key and verify that the shared secret output is the same?

Reply:
Good idea 👍